templates/resources/git_repository.md.tmpl

```diff
@@ -0,0 +1,35 @@
+---
+Title: "Git Repository Resource"
```

Comment:
Do we need a guide for git repository resource too? cc: @ishangupta-ds

Reply:
I think a guide for GitOps in general would be good since these are linked resources. @Shubbhang351